src/openrct2/ReplayManager.cpp

```diff
@@ -774,10 +774,7 @@ namespace OpenRCT2
 
         bool Compatible(ReplayRecordData& data)
         {
-            if (data.version == 1 && ReplayVersion == 2)
-                return true;
-
-            return false;
+            return data.version == 1 && ReplayVersion == 2;
```

Comment:
I have a slight preference towards the old notation, but at least this is as easy to read.

Reply:
I prefer to return single boolean expressions as oneliners, as proposed.

Reply:
Right. This case seems readable to me at least.